pkgs/tools/typesetting/polylux2pdfpc/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, rustPlatform, fetchFromGitHub, nix-update-script }:
+
+let
+  dirname = "pdfpc-extractor";
+in
+rustPlatform.buildRustPackage rec {
+  pname = "polylux2pdfpc";
+  version = "0.3.1";
```

Comment:
Shouldn't this be 0.1.0? See https://github.com/andreasKroepelin/polylux/blob/main/pdfpc-extractor/Cargo.toml#L5

Reply:
I wasn't sure what to put as the version, since the author doesn't seem to be properly releasing tags for this "sub package". I'm afraid that the code might change in the future without the version changing.

I settled on 0.3.1 because that's what the AUR package is using: https://aur.archlinux.org/packages/polylux2pdfpc (and this tool was only introduced in version 0.3.0 of the parent package: https://polylux.dev/book/changelog.html)

If you think 0.1.0 is preferable (or `unstable-2023-08-25`, the date of the last commit on that folder), I have no objections to changing it to that.

Thoughts?

Reply:
> Thoughts?

Open an upstream issue.